### ISSUE TITLE
[C/C++] Highlight of function declaration after #define

### ISSUE THREAD

**Issue by TheClams** [sublimehq/Packages#342]
If there is two consecutive #define with comment at the end of the line before some function declaration, the first one will be incorrectly highlighted as variable.function while the second one is correct as entity.name.function.

**Version**: dev Build 3111

**Test**:

```C
#define VAR0 0 // 
#define VAR1 1 // 

void test ();
void test ();
```

**Comment by wbond** [comment]
The latest version of C picks it up properly, but C++ does not. I'm working on a fix now.

**Comment by wbond** [comment]
Thanks for the bug report. This is fixed by 39f008f6021dd1c79fb45f8cc38b745efe126896.
